### DISCUSSION TITLE
Error being thrown for object Helper config

### DISCUSSION THREAD

**Discussion by neo-t-maredi** [Cyfrin/foundry-full-course-cu#1681] [Q&A]
Not sure what to do really. All paths and imports seem to work but I get this problem 
![Screenshot 2024-05-04 155632](https://github.com/Cyfrin/foundry-full-course-f23/assets/146746647/0d4e75ac-e905-4d35-ba43-f1bb1280e151)
![Screenshot 2024-05-04 160118](https://github.com/Cyfrin/foundry-full-course-f23/assets/146746647/bdeed5a1-c62e-4217-b9fb-c39516526867)
![Screenshot 2024-05-04 160131](https://github.com/Cyfrin/foundry-full-course-f23/assets/146746647/16cebd33-d464-4b3a-959e-a26d58f16e69)

**Comment by EngrPips** [comment]
Hello @neo-t-maredi, Did you run into any problems when you tried to run the script?
